content/docs/querying/functions.md

```diff
@@ -80,6 +86,55 @@ and value across all series in the input vector.
 `floor(v instant-vector)` rounds the sample values of all elements in `v` down
 to the nearest integer.
 
+## `histogram_quantile()`
+
+`histogram_quantile(φ float, b instant-vector)` calculates the φ-quantile (0 ≤ φ
```

Comment:
As said, that will lure people down the more common way of a '4-quantile' (for 25th percentile). They will get weird results once they try to run `histogram_quantile(4, series)`.

Again, I don't think we can gain anything by pretending a hard thing is simple.

In the next commit, I add a section that explains φ-quantiles, hopefully in a way everybody understands it (or let's say only people will not understand it that should better not touch histograms at all).